.github/workflows/lineage-management.yml

```
@@ -0,0 +1,224 @@
+      run: |
+        set -euo pipefail
+        DAYS_OLD="${{ github.event.inputs.days_old || '30' }}"
+        echo "Cleaning up draft lineages older than $DAYS_OLD days..."
+
+        # Note: This is a placeholder - actual cleanup would require database access
+        echo "Cleanup simulation: Would remove drafts older than $DAYS_OLD days"
+        echo "cleanup_count=0" >> $GITHUB_OUTPUT
+
+    - name: Generate health report
+      if: github.event.inputs.action == 'health-check'
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:3:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)